spec/support/helpers/fill_helper.rb

```diff
@@ -2,7 +2,10 @@ module Features
   module FillHelper
     def fill_in_datetime_field(name, with:)
       datetime_field = fill_in(name, with: with)
-      datetime_field.ancestor('.datetime-picker').sibling('.hbw-datetime-label').click
+      datetime_field.ancestor('.react-datepicker__input-container')
+                    .ancestor('.react-datepicker-wrapper')
```

Comment:
why two ancestors and not just `.ancestor('.react-datepicker__input-container .react-datepicker-wrapper')`?